### PR TITLE
Added first version of markers API

### PR DESCRIPTION
**Pull request by TS-Tino16**

### CODE REVIEW

File: apps/api/src/app/markers/dto/create-marker.input.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  @IsEnum(MarkerType)
+  @IsNotEmpty()
+  type: MarkerType;
+
+  @Field({ description: 'Marker Name' })
+  @IsNotEmpty()
+  name: string;
+
+  @Field({ description: 'Marker Name' })
+  @IsNotEmpty()
```
Comment (TS-Daniel16) [inline apps/api/src/app/markers/dto/create-marker.input.ts:17]: Should that be mandatory? Or is that only additional information besides a default color?

File: apps/api/src/app/markers/dto/update-marker.input.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  @IsEnum(MarkerType)
+  @IsNotEmpty()
+  type: MarkerType;
+
+  @Field({ description: 'Marker Name' })
+  @IsNotEmpty()
+  name: string;
+
+  @Field({ description: 'Marker Name' })
+  @IsNotEmpty()
```
Comment (TS-Daniel16) [inline apps/api/src/app/markers/dto/update-marker.input.ts:18]: Same as in the create input structure

File: apps/api/src/app/markers/entities/marker.entity.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  type: MarkerType;
+
+  @Column()
+  @Field({ description: 'Marker Name' })
+  @IsNotEmpty()
+  name: string;
+
+  @Column()
+  @Field({ description: 'Marker Color' })
+  @IsNotEmpty()
```
Comment (TS-Daniel16) [inline apps/api/src/app/markers/entities/marker.entity.ts:37]: Same as create input type

**Review by TS-Daniel16 — APPROVED** [review]

File: apps/api/src/app/markers/dto/create-marker.input.ts (hunk 1 above)
Comment (TS-Tino16) [reply apps/api/src/app/markers/dto/create-marker.input.ts]: Added default value

File: apps/api/src/app/markers/entities/marker.entity.ts (hunk 3 above)
Comment (TS-Tino16) [reply apps/api/src/app/markers/entities/marker.entity.ts]: Added default value

File: apps/api/src/app/markers/dto/update-marker.input.ts (hunk 2 above)
Comment (TS-Tino16) [reply apps/api/src/app/markers/dto/update-marker.input.ts]: Added default value

**Pull request merged**